jenkins/runlocal

```diff
@@ -15,13 +15,27 @@ export PB_SERVER_IMAGE_TAG=${USER}
 # is used by the pod.
 export PB_SERVER_IMAGE_PULL_POLICY=never
 
-# Create an RPM from the current source tree.
+# Create an RPM from the current source tree and double check it exists.
 make -C server/rpm clean rpm
+export RPM_PATH=${HOME}/rpmbuild/RPMS/noarch/pbench-server-*.rpm
+ls -ld ${RPM_PATH}
 
 source /etc/os-release
 
+if [[ ${ID} == "fedora" ]]; then
+    BASE_IMAGE=${ID}:${VERSION_ID}
+elif [[ ${ID} == "centos" ]]; then
+    BASE_IMAGE=stream${VERSION_ID%%.*}:latest
+elif [[ ${ID} == "rhel" ]]; then
+    BASE_IMAGE=ubi${VERSION_ID%%.*}:latest
```

Comment:
You solved the hard problem, but not the easy one (I think):
```
+ ((  1 != 1  ))
++ buildah from stream9:latest
Error: short-name resolution enforced but cannot prompt without a TTY
+ container=
```
The image name should be `quay.io/centos/centos:stream9`..._I think_.  There are "short names" which can be a help, e.g:
```
$ podman pull centos:stream9
Trying to pull quay.io/centos/centos:stream9...
```
So, I think line 28, at least, needs to be:
```
    BASE_IMAGE=${ID}:stream${VERSION_ID%%.*}
```
(but I haven't tried it, yet).

Reply:
> I haven't tried it, yet

It seems to be working.

Reply:
> It seems to be working.

I gotta stop saying things like that...I think this is the third time today I've cursed this PR!

```
+ buildah run centos-working-container-18 dnf install -y --nodocs /tmp/pbench-server.rpm nginx less rsyslog rsyslog-mmjsonparse
Last metadata expiration check: 0:00:13 ago on Mon Feb  6 22:43:42 2023.
Error: 
 Problem: conflicting requests
  - nothing provides python3-bcrypt needed by pbench-server-1.0.0-13ge7ed7025e.noarch
  - nothing provides python3-certifi needed by pbench-server-1.0.0-13ge7ed7025e.noarch
  - nothing provides python3-click needed by pbench-server-1.0.0-13ge7ed7025e.noarch
  - nothing provides python3-elasticsearch needed by pbench-server-1.0.0-13ge7ed7025e.noarch
  - nothing provides python3-flask needed by pbench-server-1.0.0-13ge7ed7025e.noarch
  - nothing provides python3-gunicorn needed by pbench-server-1.0.0-13ge7ed7025e.noarch
  - nothing provides python3-humanize needed by pbench-server-1.0.0-13ge7ed7025e.noarch
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
Error: while running runtime: exit status 1
```

Reply:
Both of these above problems fixed in follow-up commits.